### PR TITLE
Clarify TiIndicator required & editable properties for each targetProduct

### PR DESCRIPTION
**Pull request by chinguyen1**

### CODE REVIEW

File: api-reference/beta/resources/tiindicator.md [hunk 1]
```diff
@@ -53,6 +53,15 @@ For details about the types of indicators supported and limits on indicator coun
 |[submitTiIndicators](../api/tiindicator-submittiindicators.md)|[tiIndicator](tiindicator.md) collection|Create new tiIndicators by posting a tiIndicators collection.|
 |[updateTiIndicators](../api/tiindicator-updatetiindicators.md)|[tiIndicator](tiindicator.md) collection| Update multiple tiIndicator objects.|
 
+## Method Differences for targetProduct
```
Comment (preetikr) [inline api-reference/beta/resources/tiindicator.md:56]: Make this as : "Methods supported by each targetProduct"

File: api-reference/beta/resources/tiindicator.md [hunk 2]
```diff
@@ -53,6 +53,15 @@ For details about the types of indicators supported and limits on indicator coun
...
+| [Update tiIndicator](https://docs.microsoft.com/graph/api/tiindicator-update?view=graph-rest-beta&tabs=http) | Required fields are: `id`, `expirationDateTime`, `targetProduct`. <br> Editable fields are:  `action`, `activityGroupNames`, `additionalInformation`, `confidence`, `description`, `diamondModel`, `expirationDateTime`, `externalId`, `isActive`, `killChain`, `knownFalsePositives`, `lastReportedDateTime`, `malwareFamilyNames`, `passiveOnly`, `severity`, `tags`, `tlpLevel`. | Required fields are: `id`, `expirationDateTime`, `targetProduct`. <br> Editable fields are: `expirationDateTime`, `severity`, `description`. |
+| [Update tiIndicators](https://docs.microsoft.com/graph/api/tiindicator-updatetiindicators?view=graph-rest-beta&tabs=http)| Refer to the [Update tiIndicator](https://docs.microsoft.com/graph/api/tiindicator-update?view=graph-rest-beta&tabs=http) method above for required and editable fields for each tiIndicator.| Support for bulk update is coming soon. |
```
Comment (preetikr) [inline api-reference/beta/resources/tiindicator.md:63]: Please re-check on bulk update support

Comment (preetikr) [reply api-reference/beta/resources/tiindicator.md:63]: Also add Delete tiIndicators as well for completeness.

File: api-reference/beta/resources/tiindicator.md [hunk 3]
```diff
@@ -53,6 +53,17 @@ For details about the types of indicators supported and limits on indicator coun
...
```
Comment (preetikr) [inline api-reference/beta/resources/tiindicator.md:63]: Remove "Support for bulk update is coming soon." - Replace with [File issue](https://github.com/microsoftgraph/security-api-solutions/issues/new)

File: api-reference/beta/resources/tiindicator.md [hunk 4]
```diff
@@ -53,6 +53,17 @@ For details about the types of indicators supported and limits on indicator coun
...
+| [Delete tiIndicator](https://docs.microsoft.com/graph/api/tiindicator-delete?view=graph-rest-beta&tabs=http) | Required field is: `id`. | Required field is: `id`. |
+| [Delete tiIndicators](https://docs.microsoft.com/graph/api/tiindicator-deletetiindicators?view=graph-rest-beta&tabs=http)| Refer to the [Delete tiIndicator](https://docs.microsoft.com/graph/api/tiindicator-delete?view=graph-rest-beta&tabs=http) method above for required field for each tiIndicator.| Support for bulk update is coming soon. |
```
Comment (preetikr) [inline api-reference/beta/resources/tiindicator.md:65]: Remove "Support for bulk update is coming soon." - Replace with [File issue](https://github.com/microsoftgraph/security-api-solutions/issues/new)

Comment (chinguyen1) [reply api-reference/beta/resources/tiindicator.md]: I've removed the original text & replaced with File issue link.

**Pull request merged**